plugins/com.google.cloud.tools.eclipse.appengine.login/src/com/google/cloud/tools/eclipse/appengine/login/ui/AccountsPanel.java

```diff
@@ -0,0 +1,137 @@
+/*******************************************************************************
+ * Copyright 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+package com.google.cloud.tools.eclipse.appengine.login.ui;
+
+import com.google.cloud.tools.eclipse.appengine.login.IGoogleLoginService;
+import com.google.cloud.tools.eclipse.appengine.login.Messages;
+import com.google.cloud.tools.ide.login.Account;
+import com.google.common.annotations.VisibleForTesting;
+
+import org.eclipse.jface.dialogs.MessageDialog;
+import org.eclipse.jface.dialogs.PopupDialog;
+import org.eclipse.jface.layout.GridDataFactory;
+import org.eclipse.jface.layout.GridLayoutFactory;
+import org.eclipse.swt.SWT;
+import org.eclipse.swt.events.SelectionAdapter;
+import org.eclipse.swt.events.SelectionEvent;
+import org.eclipse.swt.graphics.Color;
+import org.eclipse.swt.widgets.Button;
+import org.eclipse.swt.widgets.Composite;
+import org.eclipse.swt.widgets.Control;
+import org.eclipse.swt.widgets.Label;
+import org.eclipse.swt.widgets.Shell;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * A panel listing all currently logged-in accounts. The panel allows adding new accounts and
+ * logging out all accounts.
+ *
+ * TODO(chanseok): move the panel to the bottom-right corner: https://github.com/GoogleCloudPlatform/google-cloud-eclipse/issues/740
+ */
+public class AccountsPanel extends PopupDialog {
+
+  private IGoogleLoginService loginService;
+
+  @VisibleForTesting Button logOutButton;
+  @VisibleForTesting List<Label> accountLabels = new ArrayList<>();
+
+  public AccountsPanel(Shell parent, IGoogleLoginService loginService) {
+    super(parent, SWT.MODELESS,
+        true /* takeFocusOnOpen */,
+        false /* persistSize */,
+        false /* persistLocation */,
+        false /* showDialogMenu */,
+        false /* showPersistActions */,
+        null /* no title area */, null /* no info text area */);
+    this.loginService = loginService;
+  }
```

Comment:
I wouldn't add that info text. In Eclipse many popups work this way, so I'd think users expect it.